README.md

```diff
@@ -57,7 +57,30 @@ You can then use our tracing dashboards to visualize the requests as traces, whi
 
 If your application uses an OpenTelemetry SDK, you can configure the application to send metrics data to Tanzu Observability using the OpenTelemetry Collector and the Wavefront proxy. Metrics data includes time series, counters, and histograms. When the data is in Tanzu Observability, you can use charts and dashboards to visualize the data and create alerts.
 
-Here's how it works:
+### Directly Send Data Using the Wavefront Proxy - (Recommended)
```

Comment:
Should we rewrite the paragraph directly above?  It seems to state that the open telemetry collector and wavefront proxy are needed, but really only the wavefront proxy is needed.

Reply:
Good catch